moped-database/migrations/1668201314245_alter_table_public_feature_drawn_lines_alter_column_render_type/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."feature_drawn_lines" alter column "render_type" set default 'line';
+alter table "public"."feature_drawn_points" alter column "render_type" set default 'point';
+alter table "public"."feature_intersections" alter column "render_type" set default 'point';
+alter table "public"."feature_signals" alter column "render_type" set default 'point';
+alter table "public"."feature_street_segments" alter column "render_type" set default 'line';
```

Comment:
Looking for thoughts about these default values for `render_type` set in the DB instead of on the front end. These values aren't consistent in the current schema either. Some tables use `Point` and `Line` while others use `line` and `point`, and then the drawn features use `LineString` and `Point` too.

Reply:
These are naive questions -- not rhetorical at all. I'm not sure I don't think I have the whole history of this end-to-end in my head. 

Is the `render_type` column vestigial from the previous map invocation? Can you get the default dimensionality hint from `moped_components.line_representation`, or even better, we could use `ST_GeometryType(geography::geometry)`. For example, `select ST_GeometryType(geography::geometry) from feature_intersections limit 1;` returns `ST_MultiPoint`, which reveals the true type of geometry/geography that can be stored in the field. We could add the output of this function to the geography view, perhaps?

That final method of inspecting the geography field itself feels the most "normal" but I totally observe that it may be more trouble than it's worth, and it's not exactly hyper-readable either. 

I'm curious what you think -- but to your question -- I think what you've done in this migration is totally fine if it's working for you. And I agree that unification of them is 💯, which is why I started thinking about how we might be able to eliminate having to track it at all, so it can't go out of sync.

Reply:
I'm looking at the code again, and I don't use `render_type` anywhere. I was going to use it to style points/lines using [Mapbox expressions](https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/), but I ran into a wall there with `visibility` not supporting expressions. 

All of the style differentiation is handled by `line_representation` (which is captured from the `moped_components` table on create), and I agree that it would be great to move to something derived from the actual geometry - that is a great idea @frankhereford. 👍 

So, we could probably drop it, and this migration really doesn't serve any purpose.